### PR TITLE
Updated the sites favicon to the hackfest logo

### PR DESCRIPTION
**Pull request by NnajiChimuanya**
This solves #6 issue

### CODE REVIEW

**Comment by NnajiChimuanya** [comment]
@amkayondo  done

**Review by HassanBahati — APPROVED** [review]

**Comment by HassanBahati** [comment]
Looks good to me!

**Comment by HassanBahati** [comment]
@NnajiChimuanya thank you for contributing :tada:

File: components/Layout.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default function Layout({ children, title }) {
           {title ? title + " - hacktoberfest uganda" : "hacktoberfest uganda"}
         </title>
         <meta name="description" content="HacktoberFest Uganda Website" />
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" href="/hack.jpg" />
```
Comment (HassanBahati) [inline components/Layout.js:13]: @NnajiChimuanya we could change the image extension from `.jpg` to `.ico`

**Comment by HassanBahati** [comment]
@NnajiChimuanya thank you for the contribution :tada:

**Pull request merged**